resources/views/review.blade.php

```diff
@@ -1,3 +1,58 @@
-<x-layout>
-    <h1>Review</h1>
+@php
+    $formattedDate = formatDate($review->created_at);
+    $backLabel = request('from') === 'movie' ? 'Back to movie' : (request('from') === 'profile' ? 'Back to profile' : 'Back');
+    $backHref = url()->previous();
+@endphp
+
+<x-layout class="pt-1 sm:pt-10">
+    <div class="flex items-start justify-between gap-6">
+        <x-section-header.back-link
+            :title="$review->movie->title"
+            extraLabel="Review"
+            href="{{ $backHref }}"
+            backLabel="{{ $backLabel }}"
+        />
+        <x-button
+            variant="icon"
+            srLabel="Open review menu"
+            class="hidden sm:block"
+        >
+            <x-lucide-ellipsis-vertical
+                class="size-6 cursor-pointer text-slate-50"
+            />
+        </x-button>
+    </div>
+    <div class="relative mt-3 flex items-center gap-4">
+        <x-poster
+            class="w-32 sm:w-40"
+            src="{{ $review->movie->cover_image }}"
+        />
+        <x-button
+            variant="icon"
+            srLabel="Open review menu"
+            class="absolute top-0 right-0 block sm:hidden"
+        >
+            <x-lucide-ellipsis-vertical
+                class="size-6 cursor-pointer text-slate-50 sm:hidden"
+            />
+        </x-button>
+
+        <div class="flex w-full flex-col gap-3">
+            <x-rating size="lg" :rating="$review->rating" />
+
+            <div
+                class="flex items-center justify-between sm:flex-col sm:items-start sm:gap-2"
```

Comment:
```suggestion
                class="flex items-center flex-wrap gap-2 justify-between sm:flex-col sm:items-start"
```
in case the username + date gets too long

https://github.com/user-attachments/assets/55e90741-e033-46b6-807d-44208d1131bd

Reply:
Did not see that, good catch!